addons/mail/tests/test_link_preview.py

```diff
@@ -111,6 +111,7 @@ def test_get_link_preview_from_url(self):
                 self.assertEqual(preview, expected)
 
     def test_link_preview(self):
+        self.env['bus.bus'].sudo().search([]).unlink()
```

Comment:
(consistency of doing things) `_reset_bus()` which does the same is used inside the same test